egs2/TEMPLATE/svs1/scripts/audio/format_scp.sh

```diff
@@ -29,6 +29,7 @@ EOF
 )
```

Comment:
Given that we already have format_wav_scp.sh, how about we just initiate a format_musicxml_scp.sh directly. It could be more clear for those users adapting from other tasks

Reply:
For now, **format_wav_scp.py** and **format_xml_scp.sh** are both called in `scripts/audio/format_scp.sh` (just like what we have in Muskit). Would it be better if we split format_scp.sh into format_wav_scp.sh and format_xml_scp.sh under scripts/audio, or just maintain the status quo?

Reply:
I would suggest separating them and using the format_wav_scp.{sh, py} from the ASR template. The major reason here is to take benefit from the timely update to those scripts if there are issues.

Reply:
format_scp.sh has been split into format_wav_scp.sh and format_xml_scp.sh.